yarn-project/noir-bb-bench/src/serve.ts

```diff
@@ -89,7 +89,7 @@ function overrideCreateDebugLog(addLogMessage: (message: string) => void): void
   const originalDebugLog = logger.debug.bind(logger);
 
   // Override the createDebug log function
-  logger.debug = (...args: any[]) => {
+  logger.debug = ((...args: Parameters<typeof originalDebugLog>) => {
```

Comment:
This change was needed since this project was not properly building apparently, and it started causing issues when added to the tsconfig references.